### ISSUE TITLE
Basic description of worker-loader's purpose

### ISSUE THREAD

**Issue by davydog187** [webpack-contrib/worker-loader#63]
I created an issue here https://github.com/webpack/webpack.js.org/issues/1161, but was asked to add to individual projects

I really love the documentation improvements that came with the new webpack 2 website. However, I find it frustrating that some of the promoted plugins/loaders on the webpack website do not have simple descriptions of their purpose.

For example, https://github.com/webpack-contrib/worker-loader states

> Worker loader for Webpack.

What is a Worker Loader? Why might I want it?

There is a lot of implicit knowledge that we're expecting developers to have. This makes it hard and intimidating for new users to start using the platform. My suggestion is to make sure that, at a minimum, worker-loader add an easily understandable description of its purpose, such that a layman could make sense of their intended use.

**Comment by TrySound** [comment]
Hi. Good idea. I'll appreciate a pull request with docs improvements.

**Comment by davydog187** [comment]
I would love to @TrySound, but I'm still unsure of what `worker-loader`'s function is. Even after reading the example code, its hard to decipher what its purpose is.

Maybe here is a good place to discuss what the description could be?

**Comment by TrySound** [comment]
Yeah. Let's do this. Do you know what is web worker? It's a code which can be run in different thread to not lock main one with big computations.

So usually it can be run like this
```js
var myWorker = new window.Worker('worker.js');
myWorker.onmessage = function(e) {
  console.log('Message received from worker');
}
```

But for this we should prepare worker.js and place it in folder near html.
worker-loader let's you just to import worker constructor and place it as usual chunk depending on configuration
```js
import MyWorker from 'worker-loader!../ww/worker.js';
var myWorker = new MyWorker();
myWorker.onmessage = function(e) {
  console.log('Message received from worker');
}
```

Also there's inline mode which can place your worker inside bundle via string with or without chunks as a fallback.

**Comment by davydog187** [comment]
Looks like this was addressed by #65. Thanks
